Source/Helpers/MessageSender.swift

```diff
@@ -159,4 +189,18 @@ public class MessageSender {
             }
         }
     }
+
+    public func send(newChannel: Message.NewChannel, date: Date, channel: Storage.Channel, completion: @escaping (Error?) -> Void) {
```

Comment:
Why do we need this func? We can just use sync sendService from startRatchetChannel

Reply:
Function become **public**:
```swift
MessageSender.public func sendService(message: Message...)
```